.storybook/main.ts

```diff
@@ -14,6 +14,9 @@ export const framework: StorybookConfig['framework'] = '@storybook/react';
 
 export const staticDirs: StorybookConfig['staticDirs'] = ['./public'];
 
+const path = require('path');
```

Comment:
Since this is TypeScript now, this should use
```typescript
import * as path from 'path';
```
Or, better yet:
```typescript
import {resolve} from 'path';
```
- [ ] named import instead of `require`

Reply:
resolved